go.mod

```diff
@@ -22,7 +22,6 @@ require (
 	github.com/juju/cmd/v3 v3.0.14
 	github.com/juju/collections v1.0.4
 	github.com/juju/errors v1.0.0
-	github.com/juju/names/v4 v4.0.0-20220207005702-9c6532a52823
```

Comment:
Looks like we mistakenly started using names/v4 instead v5 somewhere. I think we need to use just v5